src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1851,7 +1891,6 @@
     </Field>
     <Field Name="Value" Type="ExpressionSyntax"/>
   </Node>
-
```

Comment:
It is not clear why the empty line was removed. Please keep it, it makes the file easier to read.

Reply:
Tracked in https://github.com/dotnet/roslyn/issues/23222